src/puppetlabs/puppetdb/config.clj

```diff
@@ -156,7 +156,7 @@
    :subname     (format "file:%s;hsqldb.tx=mvcc;sql.syntax_pgs=true" (io/file vardir "db"))})
 
 (defn validate-db-settings
-  "Throws a {:type ::configuration-error :message m} exception
+  "Throws a {:type ::cli-error :message m} exception
```

Comment:
I'm not sure it's important, but I'd used a separate configuration-error because I wasn't sure that a config file error should be treated the same as a cli-error, and because Russell asked whether or not the TK cli-error and cli-help exceptions were actually a public interface.